django/db/backends/sqlite3/creation.py

```diff
@@ -4,11 +4,15 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.db.backends.base.creation import BaseDatabaseCreation
+from django.utils.encoding import force_text
 from django.utils.six.moves import input
 
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    def is_database_in_memory(self, database_name):
```

Comment:
I can't make a meaningful difference between this name and the other `is_in_memory_db` method. Do you intentionally name the methods differently? Should we make it a `@staticmethod` since `self` isn't used? Prefer single quotes for the strings.